pkg/provider/provider.go

```diff
@@ -101,12 +99,6 @@ func (p *ExoscaleProvider) Schema(ctx context.Context, req provider.SchemaReques
 					"Timeout in seconds for waiting on compute resources to become available (by default: %.0f)",
 					config.DefaultTimeout.Seconds()),
 			},
-			GZIPUserDataAttrName: schema.BoolAttribute{
-				Optional: true,
-				MarkdownDescription: fmt.Sprintf(
-					"Defines if the user-data of compute instances should be gzipped (by default: %t)",
-					config.DefaultGzipUserData),
-			},
```

Comment:
Perhaps we should keep the attribute but make it `no-op` and deprecated instead? Just so it does not break any existing configuration.

Reply:
This attribute hasn't any impact on the provider behavior even with currently released versions.

I'm more in favor of removing it completely since any broken configuration could be fixed by just removing this optional attribute.

Reply:
Lets do that and add note about in `CHANGELOG.md`.